### ISSUE TITLE
Requires credentials for every time running the code

### ISSUE THREAD

**Issue by Dhina17** [tdlight-team/tdlight-java#110]
Expected behaviour:
- Run the code for once, do the login, take the tdlib db. 
- I could reuse the login tdlib db to bypass the authentication process. I have to login again to use the client.

Current behaviour:
I can't reuse the previous credentials. whenever I run the code, I have to log in manually to use the client.

**Comment by s0m31-hub** [comment]
```java
var sessionPath = Paths.get("example-tdlight-session");
settings.setDatabaseDirectoryPath(sessionPath.resolve("data"));
settings.setDownloadedFilesDirectoryPath(sessionPath.resolve("downloads"));
client = new SimpleTelegramClient(settings);
``` (from https://github.com/tdlight-team/tdlight-java/blob/master/example/src/main/java/it/tdlight/example/Example.java)

**Comment by Dhina17** [comment]
> ```java
> var sessionPath = Paths.get("example-tdlight-session");
> settings.setDatabaseDirectoryPath(sessionPath.resolve("data"));
> settings.setDownloadedFilesDirectoryPath(sessionPath.resolve("downloads"));
> client = new SimpleTelegramClient(settings);
> ```
> 
> (from https://github.com/tdlight-team/tdlight-java/blob/master/example/src/main/java/it/tdlight/example/Example.java)

I followed the same

**Comment by s0m31-hub** [comment]
> > ```java
> > var sessionPath = Paths.get("example-tdlight-session");
> > settings.setDatabaseDirectoryPath(sessionPath.resolve("data"));
> > settings.setDownloadedFilesDirectoryPath(sessionPath.resolve("downloads"));
> > client = new SimpleTelegramClient(settings);
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > (from https://github.com/tdlight-team/tdlight-java/blob/master/example/src/main/java/it/tdlight/example/Example.java)
> 
> I followed the same

that's strange. Are you sure that you are doing it right? That your jvm has permission to read/write into that folder

**Comment by Dhina17** [comment]
> > > ```java
> > > var sessionPath = Paths.get("example-tdlight-session");
> > > settings.setDatabaseDirectoryPath(sessionPath.resolve("data"));
> > > settings.setDownloadedFilesDirectoryPath(sessionPath.resolve("downloads"));
> > > client = new SimpleTelegramClient(settings);
> > > ```
> > > 
> > > 
> > >     
> > >       
> > >     
> > > 
> > >       
> > >     
> > > 
> > >     
> > >   
> > > (from https://github.com/tdlight-team/tdlight-java/blob/master/example/src/main/java/it/tdlight/example/Example.java)
> > 
> > 
> > I followed the same
> 
> that's strange. Are you sure that you are doing it right? That your jvm has permission to read/write into that folder
Yes I guess the problem is with the SimpleTelegramClient impl. Let me cross check once again
